src/graph/test/SchemaTest.cpp

```diff
@@ -150,16 +126,17 @@ TEST_F(SchemaTest, metaCommunication) {
         ASSERT_TRUE(verifyResult(resp, expected));
     }
     {
+        // Test the tag has been existed.
         cpp2::ExecutionResponse resp;
-        std::string query = "CREATE TAG upper(name string, EMAIL string, "
+        std::string query = "CREATE TAG person(name string, EMAIL string, "
                             "age int, gender string, row_timestamp timestamp)";
         auto code = client->execute(query, resp);
-        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+        ASSERT_EQ(cpp2::ErrorCode::E_EXECUTION_ERROR, code);
     }
     sleep(FLAGS_load_data_interval_secs + 1);
     {
         cpp2::ExecutionResponse resp;
-        std::string query = "DESCRIBE TAG upper";
+        std::string query = "DESCRIBE TAG person";
```

Comment:
From the original version of line 130 to here, is testing unreversed keyword.
Please recover

Reply:
good ~